doc/user_manual/runInfo.tex

```diff
@@ -68,6 +68,16 @@ \subsection{RunInfo: Input of Calculation Flow}
 %
 \default{1}
 
+\item \xmlNode{maxQueueSize}, \xmlDesc{integer, optional field},
+  number of samples for RAVEN to queue.  RAVEN will request and prepare inputs
+  while other inputs are being evaluated, up to the \xmlNode{maxQueueSize}.  By
+  default, this is equal to the \xmlNode{batchSize}.  That is, for a batch size of 24,
+  at most 24 evaluations will be performed in parallel, with 24 in queue prepared to run.
+  Increasing this number will decrease the likelihood RAVEN is idle; however, increasing
+  it too much may be memory intensive and front-load the sampling significantly.
+%
+\default{1}
```

Comment:
default is the ```batchSize```, remove this line.

Reply:
I did think about this, and used "1" since if neither `batchSize` nor `maxQueueSize` are defined, then it defaults to "1".  I could change this to `batchSize` though.